benchmark/Project.toml

```diff
@@ -0,0 +1,4 @@
+[deps]
```

Comment:
From my first sight, it's not so intuitive where this file is used. I'm imagining something like 

```diff
- julia -e 'using BenchmarkCI; BenchmarkCI.displayjudgement()'
+ julia --project=benchmark -e 'using BenchmarkCI; BenchmarkCI.displayjudgement()'
```